pkg/search/options.go

```diff
@@ -263,7 +263,8 @@ var (
 	SubjectName = newFieldLabel("Subject")
 
 	// General
-	CreatedTime = newFieldLabel("Created Time")
+	CreatedTime    = newFieldLabel("Created Time")
+	DiscoveredTime = newFieldLabel("Discovered Time")
```

Comment:
I recommend making the search name more explicit, as in `XYZ discovered time`, to make it future-proof.

The search query will get indeterministic results if this label is used in two protos that are connected, which then has to be addressed through changes to the data model and migration.

You will see some legacy generic names that have been around because, admittedly, we have procrastinated changes until there is an issue, and changed only the concerning ones.